src/transformers/pipelines.py

```diff
@@ -1459,6 +1540,12 @@ def __call__(
             "tokenizer": ("t5-base", {"use_fast": False}),
         },
     },
+    "text_generation": {
```

Comment:
rename to kebab case:
```suggestion
    "text-generation": {
```